docs/changehistory/NextVersion.md

```diff
@@ -3,3 +3,8 @@ publish: false
 ---
 # NextVersion
 
+### @bentley/bentleyjs-core
```

Comment:
Since this is a release branch, any updates that would normally go into the `NextVersion.md` should be put directly in `2.19.0.md` and `index.md`.

This file isn't ever merged with those files, or replace them, like it does when we're on `master`.